tests/container-scripts/setup-validators.sh

```diff
@@ -8,9 +8,25 @@ CHAIN_ID="althea_417834-1"
 NODES=$1
 
 STAKING_TOKEN="aalthea"
-ALLOCATION="1000000000000000000000000${STAKING_TOKEN},1000000000000footoken"
+ALLOCATION="1000000000000000000000000${STAKING_TOKEN},1000000000000ufootoken"
  DELEGATION="500000000000000000000000${STAKING_TOKEN}"
 
```

Comment:
Deriving usable EVM accounts from cosmos style mnemonics is very tricky, so I opted to create some EVM users here and give them a genesis allocation, replacing what hardhat does on geth in the gravity repo.